wdio.conf.js

```diff
@@ -1,17 +1,32 @@
 /* eslint-disable import/no-extraneous-dependencies */
```

Comment:
We were so close to being able to eliminate wdio config from our repos, any way this could be moved to terra-toolkit?